src/command.rs

```diff
@@ -111,18 +138,20 @@ pub unsafe extern "C" fn wgpu_render_pass_destroy(pass: id::RenderPassId) {
 pub unsafe extern "C" fn wgpu_command_encoder_begin_compute_pass(
     encoder_id: id::CommandEncoderId,
     _desc: Option<&wgc::command::ComputePassDescriptor>,
-) -> id::ComputePassId {
-    let pass = wgc::command::RawPass::new_compute(encoder_id);
+) -> *mut ComputePass {
+    let pass = wgc::command::ComputePass::new(encoder_id);
     Box::into_raw(Box::new(pass))
 }
 
 #[no_mangle]
-pub unsafe extern "C" fn wgpu_compute_pass_end_pass(pass_id: id::ComputePassId) {
-    let (pass_data, encoder_id) = Box::from_raw(pass_id).finish_compute();
-    gfx_select!(encoder_id => GLOBAL.command_encoder_run_compute_pass(encoder_id, &pass_data))
+pub unsafe extern "C" fn wgpu_compute_pass_end_pass(pass: &mut ComputePass) {
```

Comment:
same here, we shouldn't take it as `&mut ComputePass`, but instead as `*mut ComputePass`